mlflow/utils/mlflow_tags.py

```diff
@@ -24,6 +24,8 @@
 MLFLOW_DATABRICKS_SHELL_JOB_ID = "mlflow.databricks.shellJobID"
 MLFLOW_DATABRICKS_SHELL_JOB_RUN_ID = "mlflow.databricks.shellJobRunID"
 
+MLFLOW_EXECUTION_MODE = "mlflow.backendMode"
```

Comment:
Should this be just `mlflow.project.backend`?

Reply:
yes, we can change it.

Reply:
Done